Update crossterm and console to latest version

dnaka91 opened the pull request:
The title pretty much says it all. Just bumping `crossterm` and `console` to the latest version﻿

mikaelmello commented:
Took a quick look and it seems crossterm has no breaking changes, but console has two. Did you check if they might affect us? I don't think it does, but a second eye would be good.

Also, could you please update the changelog to make this changes explicit?

Thanks

dnaka91 commented:
Oh sorry, I forgot to update the changelog. Will do so tomorrow.
I tried a few samples with both `crossterm` and `console` enabled and they worked fine. Will have a closer look at what changed in `console` as I don't use it myself (`crossterm` instead).

dnaka91 commented:
Sorry, have totally forgotten about this one 😅.
I rebased on `main`, added a changelog entry, and the changes in `console` don't seem to affect the library.
